pkg/client/response.go

```diff
@@ -44,3 +44,8 @@ func (r *Response) Bytes() []byte {
 func (r *Response) String() string {
 	return string(r.body)
 }
+
+// Header returns the header value.
```

Comment:
What will this return if there is no such header? Answer that in the documentation of the method.

Reply:
the `Get` function documentation says an empty string will be returned in such case

Reply:
I'll add it to the documentation